fix php warning:

[18-Dec-2009 23:10:11] PHP Warning:  register_shutdown_function(): Invalid shutdown callback 'Array' passed in /usr/share/horde3/lib/Horde/Kolab/Storage/List.php on line 115

$session->query('kolab_folderlist') returns a stored Kolab_List, not an Array.

--- a/horde-webmail/lib/Horde/Kolab/Storage/List.php	2010-07-06 14:14:30.000000000 +0200
+++ b/horde-webmail/lib/Horde/Kolab/Storage/List.php	2010-07-06 14:38:56.000000000 +0200
@@ -107,15 +107,15 @@
             $list = $session->query('kolab_folderlist');
         }
 
-        if (empty($list[Auth::getAuth()]) || $destruct) {
-            $list[Auth::getAuth()] = new Kolab_List();
+        if (empty($list) || $destruct) {
+            $list = new Kolab_List();
         }
 
         if (!empty($GLOBALS['conf']['kolab']['imap']['cache_folders'])) {
             register_shutdown_function(array(&$list, 'shutdown'));
         }
 
-        return $list[Auth::getAuth()];
+        return $list;
     }
 
     /**
